node_modules/oae-core/creatediscussion/js/creatediscussion.js

```diff
@@ -104,7 +104,12 @@ define(['jquery', 'oae.core'], function ($, oae) {
                 // Update visibility for discussion
                 visibility = data.visibility;
 
-                members = _.without(data.shared, oae.data.me.id);
+                members = _.chain(data.selectedPrincipalItems)
```

Comment:
If `selectedPrincipalItems` is an array of objects (as we pluck `shareId` from it), the filter doesn't make sense as it compares an object to the user id.